### PR TITLE
Update bundle_size.mdx for excluding pdfjs-dist optional dependency docs

### PR DESCRIPTION
**Pull request by abhimskywalker**
The current fix didn't work, but this updated fix did work for me. Hence proposing this as another solution.

### CODE REVIEW

File: docs/pages/common_issues/bundle_size.mdx [hunk 1]
```diff
@@ -33,7 +33,17 @@ eg: `--arch=arm64 --platform=linux --target=18 --libc=glibc`
 
 ##### pdfjs
 
-If you need to use pdfjs, you should install it with `npm i pdfjs-dist--no-optional` because the optional dep: `canvas` takes about 180MB.
+- If you need to use pdfjs, you should install it with `npm i pdfjs-dist--no-optional` because the optional dep: `canvas` takes about 180MB.
+
+- If the above dosn't work (or gives some compilation errors) you can try:
```
Comment (khuezy) [inline docs/pages/common_issues/bundle_size.mdx:38]:
```suggestion
- If the above doesn't work (or gives some compilation errors) you can try:
```

File: docs/pages/common_issues/bundle_size.mdx [hunk 2]
```diff
@@ -33,7 +33,17 @@ eg: `--arch=arm64 --platform=linux --target=18 --libc=glibc`
...
+
+```js
+  experimental: {
+    outputFileTracingExcludes: {
+      "*": ["node_modules/canvas"],
```
Comment (khuezy) [inline docs/pages/common_issues/bundle_size.mdx:43]: @revmischa was having build issues w/ the latest nextjs causing infinite loop. 🤷

Comment (revmischa) [reply docs/pages/common_issues/bundle_size.mdx:43]: I filed an issue https://github.com/vercel/next.js/issues/60301

Comment (revmischa) [reply docs/pages/common_issues/bundle_size.mdx:43]: People are reporting that updating sharp to v0.33.1 fixes this
Is this version specifier something that is inside open-next?

Comment (khuezy) [reply docs/pages/common_issues/bundle_size.mdx:43]: https://github.com/sst/open-next/blob/main/packages/open-next/src/build.ts#L370

Comment (khuezy) [reply docs/pages/common_issues/bundle_size.mdx:43]: @abhimskywalker do you mind updating the sharp version to the latest `0.33.2` and see if that fixes the issues?
https://github.com/lovell/sharp/releases/tag/v0.33.2

**Pull request merged**